en_partut-ud-train.conllu

```diff
@@ -23679,8 +23679,8 @@
 22	second	second	NOUN	S	Number=Sing	18	advcl	_	_
 23	and	and	CCONJ	CC	_	24	cc	_	_
 24	third	third	PRON	NO	NumType=Ord|PronType=Ind	22	conj	_	_
-25	as	as	ADV	B	_	24	advmod	_	_
-26	well	well	ADV	B	_	25	fixed	_	SpaceAfter=No
+25	as	as	ADV	B	ExtPos=ADV	24	advmod	_	_
```

Comment:
I would favor the reading where "as well" modifies "second and third", and therefore "as" attaches to "second".